snuba/datasets/events.py

```diff
@@ -259,16 +246,21 @@ def __init__(self):
         self.__promoted_context_columns = promoted_context_columns
         self.__required_columns = required_columns
 
+        self.__tags_processor = TagColumnProcessor(
```

Comment:
The alternative would have been to create a parent class, I decided to avoid that because there would not be a sound way to make a parent class expose the tag functions by implementing column_expr, since there is already a parent class of event implementing it (TimeSeriesEvent)